engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/expression/ExpressionEvaluator.scala

```diff
@@ -28,7 +28,8 @@ object ExpressionEvaluator {
   def optimizedEvaluator(globalVariablesPreparer: GlobalVariablesPreparer,
                          listeners: Seq[ProcessListener],
                          metaData: MetaData,
-                         services: Map[String, ObjectWithMethodDef]): ExpressionEvaluator = {
+                         services: Map[String, ObjectWithMethodDef])
+                        (implicit defaultAsyncValue: DefaultAsyncInterpretationValue): ExpressionEvaluator = {
```

Comment:
let's add TODO to remove this after getting rid of lazy values? Otherwise we'll forget about it :)